feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.spec.ts

```diff
@@ -68,4 +88,76 @@ describe('ConfigAttributeDropDownComponent', () => {
       })
     );
   });
+
+  it('should call emit of selectionChange onHandleQuantity', () => {
```

Comment:
except the first 2 tests, all other tests can be moved to its parent.

Reply:
Picked!

Reply:
Done!